tools/autodeployment/deploy.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+if [ $1 = 'production']
+then
+  DOCKER_FILE=docker-compose-production.yml
+else
+  DOCKER_FILE=docker-compose-staging.yml
+fi
```

Comment:
Here you could also include something like:
```sh
then
  DOCKER_FILE=docker-compose-production.yml
  ENV_FILE=env.production
...
```
and then use ENV_FILE later instead of having to pass a second argument to the script.

Reply:
I like this idea.

Reply:
thank you for the suggestion